src/driver.ts

```diff
@@ -192,6 +192,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
       workspaceRootFolderName: path.basename(ws_root),
       generator: this.generatorName || 'null',
       userHome: user_dir,
+      buildKit: this._kit!.name,
```

Comment:
Will have to look closer here that the non-`null` assertion won't cause any trouble.